Cargo.lock

```diff
@@ -197,13 +197,12 @@ dependencies = [
 [[package]]
 name = "finalfusion"
 version = "0.5.0"
+source = "git+https://github.com/finalfusion/finalfusion-rust.git#b5a0b1f3d8dda9d2b5abc13710cd79f54ff29ce4"
```

Comment:
Changes to `Cargo.lock` should not be part of the pull request. It is unrelated to the addition of `ff-anology`.